sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -183,6 +192,15 @@ export class FormTrainingClient {
     }
   }
 
+
+  /**
+   * Creates an instance of {@link FormTrainingClient} to perform training operations
+   * and to manage trained custom form models.
+   */
+  public getFormRecognizerClient(): FormRecognizerClient {
+    return new FormRecognizerClient(this.endpointUrl, this.credential);
```

Comment:
not sure how this works in JS, but if the user created the training client with custom pipelineOptions, are those passed to the FormRecognizerClient so that they share the same options?

Reply:
Not shared in current code.  I will look into sharing the pipeline or pipeline options.  Logged https://github.com/azure/azure-sdk-for-js/issues/9119